### PR TITLE
Update CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by TheButlah**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 # How to build
 
+## First time steps
```
Comment (TheButlah) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -5,11 +5,12 @@
 - Install [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=nl-nl&gl=nl) (optional)
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install docker: `sudo apt install docker`
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
- Install Docker: https://docs.docker.com/engine/install/

```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -5,11 +5,12 @@
 - Install [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=nl-nl&gl=nl) (optional)
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install docker: `sudo apt install docker`
 - download the included file docker.sh (start this file before starting each time!)
 - Clone the `Core` [repository](https://github.com/XB15/core)
-- Navigate to the core in your WSL window (`cd/../../core`)
-- Execute command: `docker run -it -v $PWD:/app mirror bash`
+- Navigate to the core in your WSL window: `cd/../../core`
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
- Navigate to the core in your WSL window: `cd ../../core`

```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -5,11 +5,14 @@
...
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install Docker: https://docs.docker.com/engine/install/
+
 - download the included file docker.sh (start this file before starting each time!)
 - Clone the `Core` [repository](https://github.com/XB15/core)
-- Navigate to the core in your WSL window (`cd/../../core`)
-- Execute command: `docker run -it -v $PWD:/app mirror bash`
+- Navigate to the core in your WSL window: `cd ../../core`
+
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -5,11 +5,14 @@
 - Install [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=nl-nl&gl=nl) (optional)
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install Docker: https://docs.docker.com/engine/install/
+
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -5,11 +5,12 @@
 - Install [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=nl-nl&gl=nl) (optional)
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install Docker: https://docs.docker.com/engine/install/
 - download the included file docker.sh (start this file before starting each time!)
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
- Download the included file `docker.sh` (start this file before starting each time!)
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -5,11 +5,12 @@
 - Install [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=nl-nl&gl=nl) (optional)
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install Docker: https://docs.docker.com/engine/install/
 - download the included file docker.sh (start this file before starting each time!)
 - Clone the `Core` [repository](https://github.com/XB15/core)
-- Navigate to the core in your WSL window (`cd/../../core`)
-- Execute command: `docker run -it -v $PWD:/app mirror bash`
+- Navigate to the core in your WSL window: `cd ../../core`
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
- Navigate to the core in your WSL window: `cd core`
```

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -5,11 +5,12 @@
...
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install Docker: https://docs.docker.com/engine/install/
 - download the included file docker.sh (start this file before starting each time!)
 - Clone the `Core` [repository](https://github.com/XB15/core)
-- Navigate to the core in your WSL window (`cd/../../core`)
-- Execute command: `docker run -it -v $PWD:/app mirror bash`
+- Navigate to the core in your WSL window: `cd ../../core`
+- Build and tag the docker image: `docker build -t xb15_core .`
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
- Build and tag the Docker image: `docker build -t xb15_core .`
```

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -5,11 +5,12 @@
 - Install [Microsoft Terminal](https://apps.microsoft.com/store/detail/windows-terminal/9N0DX20HK701?hl=nl-nl&gl=nl) (optional)
 - Install a subsystem (I prefer [Ubuntu](https://apps.microsoft.com/store/detail/ubuntu-22041-lts/9PN20MSR04DW))
 - Set up your WSL enviroment with a username and pw
-- Install docker (`sudo apt install docker`)
+- Install Docker: https://docs.docker.com/engine/install/
 - download the included file docker.sh (start this file before starting each time!)
 - Clone the `Core` [repository](https://github.com/XB15/core)
```
Comment (TheDevMinerTV) [inline CONTRIBUTING.md]:
```suggestion
- Clone the `Core` [repository](https://github.com/XB15/core): `git clone https://github.com/XB15/core`
```

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

**Comment by TheButlah** [comment]
Stop making suggestions that are unrelated to my pr - just open a new pr and do your edits

**Pull request merged**